packages/nextjs/components/Header.tsx

```diff
@@ -42,15 +42,21 @@ export const Header = () => {
       <li>
         <NavLink href="/projects">Projects</NavLink>
       </li>
-      <li>
+      <li className="dark:text-white">
```

Comment:
I'm not exactly a Tailwind expert, but I think that is better to use a text color class name that is set to another color on each theme at https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/blob/main/packages/nextjs/tailwind.config.js